### DISCUSSION TITLE
Is it possible to run manual tests in Allure

### DISCUSSION THREAD

**Discussion by anataty** [allure-framework/allure2#1560] [Questions & Support] [answered]
I'd like to have auto and manual tests in one place - Allure. Is it possible?

**Comment by sphinxy** [comment]
https://docs.qameta.io/allure-testops/quickstart/qa-manual/#run-tests it also have a great manual for it, same as for autotests

**Comment by cheshi-mantu** [comment, marked as answer]
@anataty 
If we're talking about Allure TestOps, then yes. Pleasse check this documentation page: https://docs.qameta.io/allure-testops/quickstart/qa-manual/
